string input_file_name
int32 nb_waypoints_path
---
string result
